src/content/articles/4004-moore-lane.md

```diff
@@ -82,9 +82,9 @@ downloadSection: {
   # The download links and button setup for the download table.
   downloads: [{
     buttonText: "DOWNLOAD",
-    downloadUrl: "https://dpel-assets.aswf.io/4004-moore-lane/intel_moore-lane.zip",
-    size: "7.5 GB",
-    description: "4004 Moore Lane v 1.0.0",
+    downloadUrl: "https://dpel-assets.aswf.io/4004-moore-lane/intel_mooreLane_v1_2_0.zip",
```

Comment:
Apparently needs to be "intel_moorelane_...", lower-case L.  (mooreLane gave me a NoSuchKey error.)